firstSeminarAdvancedAssignment/src/main/java/sopt/org/firstSeminarAdvancedAssignment/controller/ClientController.java

```diff
@@ -0,0 +1,68 @@
+package sopt.org.firstSeminarAdvancedAssignment.controller;
+
+import sopt.org.firstSeminarAdvancedAssignment.service.ClientService;
+import sopt.org.firstSeminarAdvancedAssignment.view.InputView;
+import sopt.org.firstSeminarAdvancedAssignment.view.OutputView;
+
+public class ClientController extends Controller {
+    private final ClientService clientService = new ClientService();
+
+    public ClientController() {
+        super(new InputView(), new OutputView());
+    }
+
+    public void enter() {
+        outputView.printWelcomeMessage();
+    }
+
+    public void goOut() {
+        outputView.printExitMessage();
+    }
+
+
+
+    public void registerClient() {
+        clientService.registerClient(inputView.inputAccountNumberForClientRegister(), inputView.inputClientName(), inputView.inputBirthForClientRegister(), inputView.inputTelForClientRegister(), inputView.inputAddressForClientRegister(), inputView.inputPasswordForClientRegister());
+    }
+
+    public void getClientAccountBalance() {
+        outputView.printCheckAccountBalanceMessage();
+        int clientAmountBalance = clientService.checkAccountBalance(inputView.inputAccountNumber(), inputView.inputPassword());
+
+        outputView.printClientAmountBalance(clientAmountBalance);
+    }
+
+    public void withdrawAccountBalance() {
+        outputView.printWithdrawAccountBalanceMessage();
+
+        String accountNumber = inputView.inputAccountNumber();
+        String password = inputView.inputPassword();
+        int withdrawAccount = inputView.inputWithdrawAccount();
+
+
+        int remainAmount = clientService.withdrawFromClientAccount(accountNumber, password, withdrawAccount);
+        outputView.printRemainAmountAfterWithdrawMessage(remainAmount);
+    }
+
+    public void depositMoneyIntoAccount() {
+        outputView.printDepositIntoAccountMessage();
+
+        int nowAmount = clientService.depositIntoAccount(inputView.inputAccountNumber(), inputView.inputDepositMoney());
+        outputView.printAmountAfterDeposit(nowAmount);
+    }
+
+    public void transferMoneyToAnotherAccount() {
+
+        int remainAmount = clientService.transferMoney(inputView.inputAccountNumber(), inputView.inputPassword(), inputView.inputAnotherAccountNumber(), inputView.inputTransferMoneyAmount());
```

Comment:
```suggestion
        int remainAmount = clientService.transferMoney(
                inputView.inputAccountNumber(), 
                inputView.inputPassword(), 
                inputView.inputAnotherAccountNumber(), 
                inputView.inputTransferMoneyAmount()
            );
```
저는 아규먼트들이 많을 때 요런 식으로 하는 것이 가독성이 좋더라고요...!
요건 개인 취향이니 참고만 해주세요~ 😄

Reply:
> 저는 아규먼트들이 많을 때 요런 식으로 하는 것이 가독성이 좋더라고요...! 요건 개인 취향이니 참고만 해주세요~ 😄

좋은거 같아요!! 길어질 때 참고해서 구현해보겠습니다 감사합니다!